spec/features/user_changes_password_spec.rb

```diff
@@ -0,0 +1,32 @@
+feature "User changes password" do
+  before :each do
+    user = create(:user)
+    login_as(user)
+    visit edit_settings_path
+    click_link "Change password"
+  end
+
+  scenario "with bad info" do
+
```

Comment:
Can you please remove the newlines after the `scenario` lines and before the `end`s?